config/import.plugin.js

```diff
@@ -8,9 +8,23 @@ const constants = require("./constants");
 const extensionsKey = constants.extensionsKey;
 
 function ImportPlugin(options) {
```

Comment:
how will be the user notified if en error occurs during the execution of the git commands

Reply:
will they have their extensions loaded if there are no compatible tags due to an error in the execution of the commands?

Reply:
He will not be notified. Do you think we have to notify him for such a thing?

Reply:
Of course they will have the extensions loaded. The tags are used only for showing a message if the version of the extensions is greater than the SF version. If the git commands don't work this message won't show. Otherwise everything will work as expected